purchases/src/main/kotlin/com/revenuecat/purchases/google/usecase/BillingClientUseCase.kt

```diff
@@ -20,6 +20,7 @@ internal typealias ExecuteRequestOnUIThreadFunction = (delayInMillis: Long, onEr
 private const val MAX_RETRIES_DEFAULT = 3
 private const val RETRY_TIMER_START_MILLISECONDS = 878L // So it gets close to 15 minutes in last retry
 internal const val RETRY_TIMER_MAX_TIME_MILLISECONDS = 1000L * 60L * 15L // 15 minutes
```

Comment:
Should we also name this `RETRY_TIMER_MAX_TIME_MILLISECONDS_BACKGROUND` for consistency, or is this not always used for _just_ the background?

Reply:
Yeah I originally thought to rename that, but it's also used in some other retries which are background/foreground independent, like network errors...

But I can try to put a better name so it's not confusing 👍. I'm thinking `RETRY_TIMER_MAX_TIME_SERVICE_UNAVAILABLE_FOREGROUND` and change the type to `Duration` to avoid having the `milliseconds` suffix (still a bit of a mouthful, but this is currently only used for that specific error)

Reply:
Tried to improve it here: https://github.com/RevenueCat/purchases-android/pull/1745/commits/aa29f7eac7c8f98d0683821ea3be27d297ed7d0e

Reply:
Alright, yea good improvement! Instead of `_MILLISECONDS` we could also use `_MILLIS` but that has to be a convention we'd choose to follow.